spec/requests/admin/configs_spec.rb

```diff
@@ -89,17 +89,16 @@
         end
 
         it "enables email authentication" do
-          post "/admin/config", params: { site_config: { allow_both_email_signup_and_login: true },
+          post "/admin/config", params: { site_config: { allow_email_password_registration: true },
                                           confirmation: confirmation_message }
           expect(SiteConfig.allow_email_password_registration).to be(true)
           expect(SiteConfig.allow_email_password_login).to be(true)
         end
 
         it "disables email authentication and invite-only mode" do
-          post "/admin/config", params: { site_config: { allow_both_email_signup_and_login: false },
+          post "/admin/config", params: { site_config: { allow_email_password_registration: false },
                                           confirmation: confirmation_message }
           expect(SiteConfig.allow_email_password_registration).to be(false)
-          expect(SiteConfig.allow_email_password_login).to be(false)
           expect(SiteConfig.invite_only_mode).to be(false)
```

Comment:
I struggled with adding such a check: typically I wouldn't add tests to make sure something _didn't_ happen, but I think it's warranted here since the whole point of this PR is turning off the previous behavior. Will add!